index.html

```diff
@@ -1239,19 +754,17 @@ <h2>Privacy and Security</h2>
 requested that resource. To limit the access to the
 <a>PerformanceResourceTiming</a> interface, the <a data-cite=
 "HTML#same-origin">same origin</a> policy is enforced by default
-and certain attributes are set to zero, as described in <a href=
-"#sec-cross-origin-resources"></a>. Resource providers can
+and certain attributes are set to zero, as described in [=/HTTP fetch=]. Resource providers can
 explicitly allow all timing information to be collected for a
-resource by adding the <a>Timing-Allow-Origin</a> HTTP response
+resource by adding the [=request/timing allow failed flag|Timing Allow Origin=] HTTP response
```

Comment:
This is the wrong link. You're talking about a header but linking to a flag.

Reply:
fixed